frontend/public/components/kube-admin-notifier.jsx

```diff
@@ -16,16 +17,16 @@ export const KubeAdminNotifier = connect(userStateToProps)(({ user }) => {
   const canUpgrade = useCanClusterUpgrade();
   const username = _.get(user, 'username');
   return KUBE_ADMIN_USERNAMES.includes(username) && canUpgrade ? (
-    <div className="co-global-notification">
-      <div className="co-global-notification__content">
-        <p className="co-global-notification__text">
+    <Banner color="blue">
```

Comment:
could possibly use `status='warning'` prop here, but that would change the color from blue to yellow